extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -744,6 +792,47 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Refreshes ad slot by fetching a new creative and rendering it. This leaves
+   * the current creative displayed until the next one is ready.
+   *
+   * @param {function()} refreshEndCallback When called, this function will
+   *   restart the refresh cycle.
+   */
+  refresh(refreshEndCallback) {
+    this.isRefreshing = true;
+    this.tearDownSlot();
+    this.initiateAdRequest();
+    const promiseId = this.promiseId_;
+    this.adPromise_.then(() => {
+      if (!this.isRefreshing || promiseId != this.promiseId_) {
+        // If this refresh cycle was canceled, such as in a no-content
+        // response case, keep showing the old creative.
+        refreshEndCallback();
+        return;
+      }
+      this.togglePlaceholder(true);
+      this.destroyFrame(true);
+      // We don't want the next creative to appear too suddenly, so we
+      // show the loader for a quarter of a second before switching to
```

Comment:
I still think a quarter of a second is short... most people can't really perceive that other than a flash.  I suggest this be 1sec